### ISSUE TITLE
Generating training material - F/B reconstruction

### ISSUE THREAD

**Issue by danielvandenberg95** [MarcoForte/FBA_Matting#25]
In the paper you mention that "Levin's F,B estimation technique" was used. That paper however only gives a very brief mention of what method was used for reconstructing.

Now I'm trying to see if it would be doable to fine-tune this model to work with higher resolutions (around 1K), which means I want to generate some extra training data. Would you have any resources that could help me implement Levin's? (Or even better, how did you do it?)

**Comment by zoezhou1999** [comment]
Same ask. Recently, I used the cropped and resized fg, bg, alpha to compose images online without the F/B reconstruction and calculated the l1 and lap loss of fg and bg with alpah>0 and alpha<1, respectively. I am wondering that might make the performance worse. And I also tried the fg extension code provided in this GitHub and I found sometimes the reconstructed fg is not good enough so I dropped it.

**Comment by kartikwar** [comment]
where is the FG extension code @zoezhou1999. When I open the link in Readme it gives 404.

**Comment by 99991** [comment]
A commit in December moved the file.

This is the last version where the file still was in the same location:

https://github.com/MarcoForte/closed-form-matting/blob/622a6b74f333cb6cf9d5458a16302ab8b605c69a/solve_foreground_background.py

This is the new location:

https://github.com/MarcoForte/closed-form-matting/blob/master/closed_form_matting/solve_foreground_background.py

This related issue might be interesting for you as well:

https://github.com/MarcoForte/FBA_Matting/issues/30
